app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
```

Comment:
Following up on my previous comment here, I've changed it so the url shows /profile like Josh explains above but it seems like it the controller needs to be left as plural.